TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,24 @@
+package org.firstinspires.ftc.teamcode.subsystems.elevator;
+
+import com.arcrobotics.ftclib.controller.PIDController;
+
+public class ElevatorConstants {
```

Comment:
Finshing working on a feature means doing the entire work, not just writing the code. Those additional task in this file should be handled before merging.